src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/Microsoft.DotNet.ScenarioTests.SdkTemplateTests.csproj

```diff
@@ -5,5 +5,14 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
   </PropertyGroup>
+	<PropertyGroup Condition="$([MSBuild]::IsOSPlatform('Windows'))">
```

Comment:
I don't think we want to compile in information about the OS that the scenario tests repo is built on. Ideally we can build this repo anywhere and run on on any OS.

Reply:
Yeah, I only have it because I ended up having to use OS specific code. If we can remove it with a better terminate process then I'm all for it.